cutlery/_wordpiece.pxd

```diff
@@ -1,7 +1,15 @@
 from cython cimport size_t
 from libcpp.string cimport string
 from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector
+
+cdef extern from "wordpiece.hh":
+    cdef cppclass Piece:
+        string piece
+        bint is_initial
+        Piece(const string& piece, bint initial)
 
 cdef class WordPieceProcessor:
-    cdef unordered_map[string, size_t] initial_pieces
-    cdef unordered_map[string, size_t] continuation_pieces
+    cdef unordered_map[string, size_t] initial_piece_to_id
+    cdef unordered_map[string, size_t] continuation_piece_to_id
```

Comment:
Sorry for only figuring this out now, but I think we can use `Piece` here as well? So, we'd have

```suggestion
    cdef unordered_map[Piece, size_t] piece_to_id
```

What do you think?

Reply:
Oh yeah, that's a good point. I'll switch that too.